packages/toolkit/src/createSlice.ts

```diff
@@ -100,9 +100,6 @@ export interface CreateSliceOptions<
    * A callback that receives a *builder* object to define
    * case reducers via calls to `builder.addCase(actionCreatorOrType, reducer)`.
    * 
-   * Alternatively, a mapping from action types to action-type-specific *case reducer*
```

Comment:
Did we at this point only have the sentence, but no more examples of it?

Reply:
yeah, the sentence was left in while the syntax itself was removed

Reply:
Then let's get this merged :)